scripts/scripts.js

```diff
@@ -217,4 +220,14 @@ async function loadPage() {
   loadDelayed();
 }
 
+export function decorateInnerHrefButtonsWithArrowIcon(block) {
+  block.querySelectorAll('a').forEach((a) => {
+    const icon = domEl('calcite-icon', { icon: 'arrowRight', scale: 's' });
+    icon.style.marginInlineStart = '4px';
```

Comment:
Why not set this style via css?

Reply:
you could add it to `styles.css`, but that seems clunky. 

You add and icon here and style it in some place else.

Let me know if you're strongly opinionated about it. I can move it there.

Reply:
I'd rather do that, you can consider styles.css is to scripts.js as <block_name>.css is to <block_name>.js.